Manual.md

```diff
@@ -1272,6 +1288,229 @@ cache during the install/removal of the package
+
+During removal of the gtk+ package it deletes the `icon-theme.cache` file
+in the directories defined by the variable `gtk_iconcache_dirs`
+
+It is automatically added on packages that have `/usr/share/icons` available
+as a directory, all directories under that directory have their absolute path
+added.
+
+<a id="triggers_gtk_immodules"></a>
+#### gtk-immodules
+
+The gtk-immodules trigger is responsible for updating the IM (Input Method) modules
+file for gtk+
+
+During installation it uses `gtk-query-immodules-2.0 --update-cache` to update the
+cache file. It also removes the obsolete configuration file  `etc/gtk-2.0/gtk.immodules`
+if present.
+
+During removal of the `gtk+` package it removes the cache file which is located at
+`usr/lib/gtk-2.0/2.10.0/immodules.cache`.
+
+It is automatically added to packages that have `/usr/lib/gtk-2.0/2.10.0/immodules`
+present as a directory.
+
+<a id="triggers_gtk_pixbuf_loaders"></a>
+#### gtk-pixbuf-loaders
+
+gtk-pixbuf-loaders is the old name for the current `gdk-pixbuf-loaders` trigger and is
+in the process of being removed. it currently re-execs into `gdk-pixbuf-loaders` as a
+compatibility measure.
+
+For information about how it works refer to [gdk-pixbuf-loaders](#triggers_gdk_pixbuf_loaders)
+
+<a id="triggers_gtk3_immodules"></a>
+#### gtk3-immodules
+
+The gtk3-immodules trigger is responsible for updating the IM (Input Method) modules
+file for gtk+3
+
+During installation it uses `gtk-query-immodules-3.0 --update-cache` to update the
+cache file. It also removes the obsolete configuration file  `etc/gtk-3.0/gtk.immodules`
+if present.
+
+During removal of the `gtk+3` package it removes the cache file which is located at
+`usr/lib/gtk-3.0/3.0.0/immodules.cache`.
+
+It is automatically added to packages that have `/usr/lib/gtk-3.0/3.0.0/immodules`
+present as a directory.
+
+<a id="triggers_hwdb.d_dir"></a>
+#### hwdb.d-dir
+
+The hwdb.d-dir trigger is responsible for updating the hardware database
+
+During installation and removal it runs `usr/bin/udevadm hwdb --root=. --update`.
+
+It is installed automatically if `/usr/lib/udev/hwdb.d` is present as a directory
+on the package.
+
+<a id="triggers_info_files"></a>
+#### info-files
+
+The info-files trigger is responsible for registering and unregistering the GNU info
+files for a package.
+
+It checks the existence of the info files presented to it and if it is running under
+another architecture.
+
+During installation it uses `install-info` to register info files into
+`usr/share/info`
+
+During removal it uses `install-info --delete` to remove the info files from the
+registry located at `usr/share/info`
+
+If it is running under another architecture it tries to use the host `install-info`
+utility
+
+<a id="triggers_kernel_hooks"></a>
+#### kernel-hooks
+
+The kernel-hooks trigger is responsible for running whatever executable files are
+found for it's target.
```

Comment:
possessive it's -> its

Reply:
> The kernel-hooks trigger is responsible for running whatever executable files are
found for it's target.

So it executes anything under `usr/bin/`? Maybe it's just because I don't know void well, but I don't quite understand this hook.

Reply:
i rewrote anything, please read again and if you don't understand i'll keep trying.

Reply:
OK, I guess this sentence is a little redundant now that you introduced the below paragraph explaing that everything in /etc/kernel.d/$TARGET will be run. I got what the hook does now though.